scripts/qa_crowbarsetup.sh

```diff
@@ -1027,6 +1030,23 @@ function onadmin_allocate()
     rm -f /root/crowbartest.out
 }
 
+function configure_compute_nodes()
+{
+    echo "Configuring the compute nodes as defined by hvmix..."
+    local computenode
+    local i=0
+    for computenode in `crowbar machines list | LC_ALL=C sort | grep ^d | tail -n +2` ; do
```

Comment:
This will not work in HA mode. A node that was fetched like this may already be a pacemaker cluster node.
Please use `$nodescompute`, it has a list of the compute nodes.